### ISSUE TITLE
Working Tailwind CSS Example / Docs

### ISSUE THREAD

**Issue by totalhack** [formkit/formkit#242]
I was trying to take the the example @andrew-boyd provided [here](https://github.com/formkit/formkit/issues/63#issuecomment-1123975397) and switch it to tailwind in a project that is using the [formkit starter project](https://github.com/formkit/formkit-vue-starter-project). 

I followed the docs [here](https://formkit.com/guides/create-a-tailwind-theme) and it didn't work. I then noticed there is a slightly different approach mentioned [here](https://github.com/formkit/formkit/tree/master/packages/tailwindcss) so I switched to that since things weren't working. Only difference between those two seems to be the use of the "themes" package vs directly using @formkit/tailwindcss. 

But then I also needed tailwindcss added to package.json and needed to add `<script src="https://cdn.tailwindcss.com"></script>` to index.html to finally get things to work, but i'm not sure if that should have been necessary and I just did something wrong. 

Perhaps it would be useful to provide a complete example project utilizing tailwind with the recommended approach? Or just modify the current docs to note what else is needed to get a working example? 

Thanks.

**Comment by garylake** [comment]
I followed @andrew-boyd's tailwind theme docs and it worked a treat. 

Using Tailwind CDN is a red flag as you won't be benefitting from JIT and minimal CSS output. 

I'm just looking at my setup on current project and can share the following hints: 

- Tailwind npm installed and is running via my postcss.config.js – focus on getting tailwind running without CDN if you haven't already: https://tailwindcss.com/docs/guides/vite (this is for vite but I'm languishing on vue-cli and this mostly works the same past the vite steps at the start)

- FormKit npm installed 

- Formkit and your formkit theme included in main.js, mine looks like: 
```
// FormKit
import App from './App.vue';

// Router
import router from './router/routes';

// Stores
import { createPinia } from 'pinia'

// FormKit
import { plugin, defaultConfig } from '@formkit/vue';
import formKitTheme from './lib/FormKit/formKitThemeMVP'

// Styles
import './main.css';

const app = createApp(App);

app.use(createPinia());
app.use(router);
app.use(plugin, defaultConfig({ config: formKitThemeMVP, }));

app.mount('#app');
```
- My formkit theme I'm not going to include here but I just followed Andrew's example in his tailwind theming guide. Got a few basics working to understand it at first (e.g. red borders everywhere 😆). I didn't know where to put it so it currently lives in ./lib/FormKit – open to suggestions on where to store it btw! 


And I'm pretty sure that was it. Break it down, validate Tailwind is working properly, validate formkit is working properly, and then do a minimal single lines things in theme file to make sure it's working – maybe try the following themefile to MVP it: 

Example formKitThemeMVP theme file:
```
import { generateClasses } from '@formkit/tailwindcss'

const formkitThemeMVP = {
  classes: generateClasses({
    form: {
      form: 'border border-red-600',
    },
  }),
};
  
export default formkitThemeMVP;
```

If that works it'll put a red border on every instance of a FormKit type="form"
